src/connector.ts

```diff
@@ -134,4 +142,22 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
```

Comment:
1. I guess `ctx` stands for context?
1. Do we have types for ctx and next?
1. What is the response, when calling this endpoint? And is it formatted such that the caller can, in a structured way, decipher it?

Reply:
1. True, it is short for context. 
2. We do have types, which is e.g. `next: Koa.next` and that would be nice to add but i decided not to since the type for context: `ctx: Koa.ParameterizedContext<any, KoaRouter.IRouterParamContext<any, {}>, any>`. 
3. I guess there really shouldn't be a specific response other than maybe success or "failure".

Reply:
You could use a custom type that encapsulates the data that you need from the koa context.